src/main/groovy/org/jetbrains/intellij/PatchPluginXmlTask.groovy

```diff
@@ -98,6 +99,16 @@ class PatchPluginXmlTask extends ConventionTask {
         this.untilBuild = untilBuild
     }
 
+    @Input
+    @Optional
+    String getChangeNotes() {
+        Utils.stringInput(releaseNotes)
+    }
+
+    void setChangeNotes(Object changeNotes) {
+        this.changeNotes = changeNotes
+    }
```

Comment:
it's recommended to add one more setter like `void changeNotes(Object changeNotes)`. In this case end-user will be able to write something like

```
patchPluginXml {
  changeNotes "Something changed"
}
```